src/spdx2/agent/template/listedlicense.xml.twig

```diff
@@ -3,15 +3,15 @@
    SPDX-License-Identifier: FSFAP
 #}
 {% macro listedLicenseFull(licenseObj) %}
-<spdx:ListedLicense rdf:about="http://spdx.org/licenses/{{ licenseObj.spdxId|replace({' ': '-'})|url_encode }}">
+<spdx:ListedLicense rdf:about="{{ ( "http://spdx.org/licenses/" ~ licenseObj.spdxId)|replace({' ': '-'}) }}">
   <spdx:name>{{ licenseObj.fullName|e }}</spdx:name>
   <spdx:licenseId>{{ licenseObj.spdxId|replace({' ': '-'})|e }}</spdx:licenseId>
   <spdx:licenseText><![CDATA[
 {{ licenseObj.text|replace({'\f':''})
     |replace({']]>':']]]]><![CDATA[>'}) }}
   ]]></spdx:licenseText>
 {%~ if licenseObj.url is not empty %}
-  <rdfs:seeAlso>{{ licenseObj.url }}</rdfs:seeAlso>
+  <rdfs:seeAlso>{{ licenseObj.url|url_encode }}</rdfs:seeAlso>
```

Comment:
Also, please do this `url_encode` **only** if there is an `&` in the `licenseObj.url` otherwise other tools reading this file might break.